src/httpserver/file_io/file_input.clj

```diff
@@ -0,0 +1,17 @@
+(ns httpserver.file-io.file-input
+  (:require [clojure.java.io :as io])
+  (:import java.io.FileInputStream))
+
+(defn- partial-content [array-of-bytes start end]
+  (subvec (mapv byte array-of-bytes) start end))
+
```

Comment:
Does this comment above apply to all functions in this namespace? Just clarifying, as I'm not sure what this specific ns would contain / be used for.